### PR TITLE
ENH: Change naming convention

### PR DESCRIPTION
**Pull request by younik**
Following the discussion in https://github.com/Farama-Foundation/Minari/pull/217, we made the env_name of a dataset id, part of the namespace.
This means datasets with id like `door-human-v2` becomes now `door/human-v2`. This allows to have metadata for env groups to easily generate documentation and shared information (e.g. `ref_min_score` and `ref_max_score`).

### CODE REVIEW

**Comment by alexdavey** [comment]
Hi @younik, the PR looks great, thank you! I few minor comments but otherwise it looks good to me :)

It would be interesting to know your thoughts on whether we should put all of the D4RL datasets in their own namespace, e.g. `D4RL/door/human-v1`. This would prevent name clashes with other future datasets, including third party datasets. On the other hand, of course, it does clutter to the dataset names in a way that it not necessary at the moment.

File: docs/content/minari_cli.md [hunk 1]
```diff
@@ -158,7 +158,7 @@ Trajectories have expert data from a fine-tuned RL policy provided in the <a hre
 │ <b>Email</b>             │ rperezvicente@farama.org                  │
 │ <b>Code Permalink</b>    │ <a href="https://github.com/rodrigodelazcano/d4rl-minari-dataset-generation">https://github.com/rodrigodelazcano/...</a>   │
```
[the remark this answers is not in the document]
Comment (younik) [reply docs/content/minari_cli.md:159]: Good catch, I also need to change the metadata in remote

File: minari/data_collector/data_collector.py [hunk 2]
```diff
@@ -242,7 +242,7 @@ def create_dataset(
         """Create a Minari dataset using the data collected from stepping with a Gymnasium environment wrapped with a `DataCollector` Minari wrapper.
 
         The ``dataset_id`` parameter corresponds to the name of the dataset, with the syntax as follows:
-        ``(namespace/)(env_name-)(dataset_name)(-v(version))`` where ``env_name`` identifies the name of the environment used to generate the dataset ``dataset_name``. The `namespace` (and leading forward slash) is optional.
+        ``(namespace/)(env_name/)dataset_name(-v(version))`` where ``env_name`` identifies the name of the environment used to generate the dataset. The `namespace` is optional.
```
[the remark this answers is not in the document]
Comment (younik) [reply minari/data_collector/data_collector.py:245]: I search for `env_name` and removed it in bunch of parts. However, I thought it was a good idea to keep it in the documentation of `DataCollector`, `minari.create_dataset_from_buffers` and in dataset standards page, to suggest the user to adhere to the convention. Can you check it again and tell me if you disagree? Thanks!

**Comment by younik** [comment]
> It would be interesting to know your thoughts on whether we should put all of the D4RL datasets in their own namespace, e.g. `D4RL/door/human-v1`. This would prevent name clashes with other future datasets, including third party datasets. On the other hand, of course, it does clutter to the dataset names in a way that it not necessary at the moment.

I don't think it is necessary at the moment, but the remote grows it may become necessary.
I would postpone it for when it becomes useful, however changing the namespace later will cause problems. So yes, it is probably a good idea to do it now.

File: minari/data_collector/data_collector.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (alexdavey) [reply minari/data_collector/data_collector.py]: Great, seems like a good idea to keep it in those places :)

**Review by alexdavey — APPROVED** [review]
LGTM @younik, thanks for addressing the comments so quickly!

**Pull request merged**